core/src/plugins/kubernetes/run.ts

```diff
@@ -1120,6 +1120,8 @@ export class PodRunner extends PodRunnerParams {
           }
 
           return errorDesc
+        case "kubernetes":
+          return `Unable to start command execution. Kubernetes cluster returned error:\n${error.message}\n\nPlease check the cluster health.`
```

Comment:
I don't think cluster itself returns errors. Perhaps this should be changed to something like

```suggestion
          return `Unable to start command execution. Failed to initiate test-runner pod with error:\n${error.message}\n\nPlease check the cluster health.`
```

Reply:
Well, the underlying library function `KubeApi.createNamespacedPod(...)` can throw errors, and we always wrap 'em as `KubernetesError`. Effectively, those are always pod-creation errors.

So we can change the error message as you suggested, but remove "test-" prefix. It can be a task execution.

Reply:
Changed this in https://github.com/garden-io/garden/pull/3388/commits/0bcef414c979591eabc19668b2ad26fc4904c04f